modules/core/src/main/java/org/apache/ignite/internal/processors/cache/persistence/snapshot/IgniteSnapshotManager.java

```diff
@@ -3785,47 +3784,32 @@ public synchronized void stop() {
 
             while ((r = queue.poll()) != null)
                 r.acceptException(new IgniteException(SNP_NODE_STOPPING_ERR_MSG));
-
```

Comment:
Suggestion: let's introduce single `Exception rre = new IgniteException(SNP_NODE_STOPPING_ERR_MSG);` before `if (active != null)` and use it instead of many the same `new IgniteException(SNP_NODE_STOPPING_ERR_MSG)`